test/test_core.py

```diff
@@ -552,6 +552,42 @@ def test_view_definition(engine_testaccount, db_parameters):
             "DROP VIEW IF EXISTS {0}".format(test_view_name))
 
 
+def test_view_comment_reading(engine_testaccount, db_parameters):
+    """
+    Tests reading a comment from a view once it's defined
+    """
+    test_table_name = "test_table_sqlalchemy"
+    test_view_name = "testview_sqlalchemy"
+    engine_testaccount.execute("""
+CREATE OR REPLACE TABLE {0} (
+    id INTEGER,
+    name STRING
+)
+""".format(test_table_name))
+    sql = """
+CREATE OR REPLACE VIEW {0} AS
+SELECT * FROM {1} WHERE id > 10""".format(
+        test_view_name, test_table_name)
+    engine_testaccount.execute(text(sql).execution_options(
+        autocommit=True))
+    comment_text = "hello my viewing friends"
+    sql = "COMMENT ON VIEW {0} IS '{1}';".format(
+        test_view_name, comment_text)
+    engine_testaccount.execute(text(sql).execution_options(
+        autocommit=True))
+    try:
+        inspector = inspect(engine_testaccount)
+        # NOTE: sqlalchemy doesn't have a way to get view comments specifically,
+        # but the code to get table comments should work for views too
+        assert inspector.get_table_comment(test_view_name) == {'text': comment_text}
+    finally:
+        engine_testaccount.execute(
+            "DROP TABLE IF EXISTS {0}".format(test_table_name))
```

Comment:
I'm just curious why did you wrap the `sql` in text on line 571, but not here.

Reply:
@keller00 now that you mention it, that does look odd -- I had in fact just copied it from the function above. I'll update them both.